karaf/images.yml

```diff
@@ -18,16 +18,6 @@ config:
       description: "CentOS S2I Karaf builder image with OpenJDK 8"
       version:
         maven: "3.6.3"
-        jolokia: "1.6.3"
+        jolokia: "1.6.2"
```

Comment:
why downgrade ?

Reply:
![grafik](https://user-images.githubusercontent.com/2443866/82296041-e2684700-99b0-11ea-966a-3219b04b8386.png)
Version1.6.3 does not exists (yet).

Reply:
The Java part refers to 1.6.2.

Reply:
sorry, you are absolutely right, 1.6.2 is the latest (i should know it 😬 )